plugins/source/mysql/resources/plugin/plugin_test.go

```diff
@@ -123,26 +190,27 @@ func TestPlugin(t *testing.T) {
 	}
 	defer db.Close()
 
-	testTable := testdata.TestSourceTable("test_mysql_source")
+	testTable := schema.TestTable("test_mysql_source", schema.TestSourceOptions{})
 	if _, err := db.ExecContext(ctx, "DROP TABLE IF EXISTS test_mysql_source"); err != nil {
 		t.Fatal(err)
 	}
 	if err := createTable(ctx, db, testTable); err != nil {
 		t.Fatal(err)
 	}
-	data := testdata.GenTestDataV1(testTable)
+	syncTime := time.Now()
+	data := schema.GenTestData(testTable, schema.GenTestDataOptions{MaxRows: 2, SyncTime: syncTime})
 	if err := insertTable(ctx, db, testTable, data); err != nil {
 		t.Fatal(err)
 	}
 
-	otherTable := testdata.TestSourceTable("other_mysql_table")
+	otherTable := schema.TestTable("other_mysql_table", schema.TestSourceOptions{})
 	if _, err := db.ExecContext(ctx, "DROP TABLE IF EXISTS other_mysql_table"); err != nil {
 		t.Fatal(err)
 	}
 	if err := createTable(ctx, db, otherTable); err != nil {
 		t.Fatal(err)
 	}
-	otherData := testdata.GenTestDataV1(otherTable)
+	otherData := schema.GenTestData(otherTable, schema.GenTestDataOptions{MaxRows: 1, SyncTime: syncTime})
```

Comment:
does it allow for 2 rows (to test the null row)?

Reply:
We test for 2 rows a bit earlier in the code (see `data := schema.GenTestData`). `otherData` is inserted o a table that is not part of the spec tables and meant to check that we don't sync it.